.betterer.results

```diff
@@ -749,10 +749,6 @@ exports[`better eslint`] = {
     "packages/grafana-runtime/src/analytics/types.ts:5381": [
       [0, 0, 0, "Unexpected any. Specify a different type.", "0"]
     ],
-    "packages/grafana-runtime/src/components/DataSourcePicker.tsx:5381": [
-      [0, 0, 0, "Use data-testid for E2E selectors instead of aria-label", "0"],
-      [0, 0, 0, "Use data-testid for E2E selectors instead of aria-label", "1"]
-    ],
```

Comment:
pushed up a commit to resolve betterer complaints 😄

Reply:
@nmarrs the Select doesn't support `data-test-id`. It only works with `aria-label`. This is breaking units and E2Es. Let's do this in a separate PR to not disrupt this one. We're kind of rushing to have the DSPicker in place asap. Thank you so much to give it a try! 🙏